RNTester/Podfile

```diff
@@ -65,24 +64,37 @@ target 'RNTester' do
   # use_react_native!(path: "..", turbo_modules_enabled: true, fabric_enabled: true)
 end
 
+target 'RNTester' do
+  platform :ios, '9.0'
+  pods()
+end
+
+target 'RNTesterUnitTests' do
+  platform :ios, '9.0'
+  pods()
+  pod 'React-RCTTest', :path => "./RCTTest"
```

Comment:
Looks like this could be grouped in a `test_pods` method. Is this same as upstream?

Reply:
Yes, this is the same upstream: there is just a single `pods()` shared function and each test target adds React-RCTTest.   It would be a good change to have upstream and down tho